tests/integration/cocoon/test_cocoon_instruments.py

```diff
@@ -212,24 +235,39 @@ def test_load_from_data_frame_instruments_success(
         :param dict{str, str} identifier_mapping: The dictionary mapping of LUSID instrument identifiers to identifiers in the dataframe
         :param list[str] property_columns: The columns to create properties for
         :param str properties_scope: The scope to add the properties to
+        :param str | None instrument_scope: The scope to add the instruments to
         :param any expected_outcome: The expected outcome
 
         :return: None
         """
 
         data_frame = pd.read_csv(Path(__file__).parent.joinpath(file_name))
 
-        responses = cocoon.cocoon.load_from_data_frame(
-            api_factory=self.api_factory,
-            scope=scope,
-            data_frame=data_frame,
-            mapping_required=mapping_required,
-            mapping_optional=mapping_optional,
-            file_type="instruments",
-            identifier_mapping=identifier_mapping,
-            property_columns=property_columns,
-            properties_scope=properties_scope,
-        )
+        if instrument_scope == None:
+            responses = cocoon.cocoon.load_from_data_frame(
+                api_factory=self.api_factory,
+                scope=scope,
+                data_frame=data_frame,
+                mapping_required=mapping_required,
+                mapping_optional=mapping_optional,
+                file_type="instruments",
+                identifier_mapping=identifier_mapping,
+                property_columns=property_columns,
+                properties_scope=properties_scope,
+            )
+        else:
+            responses = cocoon.cocoon.load_from_data_frame(
+                api_factory=self.api_factory,
+                scope=scope,
+                data_frame=data_frame,
+                mapping_required=mapping_required,
+                mapping_optional=mapping_optional,
+                file_type="instruments",
+                identifier_mapping=identifier_mapping,
+                property_columns=property_columns,
+                properties_scope=properties_scope,
+                instrument_scope=instrument_scope,
+            )
```

Comment:
Why do we need this condition? If we pass None in as instrument_scope doesn't it default to "default" anyway?